app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -1,16 +1,70 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+# require "net/http"
+# require "sys/filesystem"
+# require "uri"
+# require "json"
+# require 'fusor/password_filter'
+# require 'fusor/deployment_logger'
+
 class Fusor::Api::V21::DeploymentsController < ApplicationController
-  before_action :set_deployment, only: [:show, :update, :destroy]
+  before_action :set_deployment, only: [:show,
+                                        :update, :destroy,
+                                        :check_mount_point,
+                                        :deploy,
+                                        :redeploy,
+                                        :validate,
+                                        :log,
+                                        :openshift_disk_space,
+                                        :compatible_cpu_families]
+
+  include Fusor::Api::V21::DiscoveredHostsMixin
```

Comment:
@isratrade OK, I'll treat this as a "todo" then and we'll handle it at the same time the front end is being updated.